test/debugger.ts

```diff
@@ -20,10 +20,16 @@
  * @module debug/debugger
  */
 
-var pjson = require('../package.json');
-var common = require('@google-cloud/common');
-var qs = require('querystring');
-var util = require('util');
+import * as commonTypes from '../src/types/common-types';
+
+var pjson = require('../../package.json');
+const common: commonTypes.Common = require('@google-cloud/common');
+// TODO: Verify these types are correct.
+const qs: { parse: (qs: any, sep?: string, eq?: string,
+  options?: { maxKeys?: number }) => any,
+  stringify: (obj: object|string|boolean|number, sep?: string,
+    eq?: string, name?: string) => string} = require('querystring');
```

Comment:
Yes, the type should be available.  There is also a `querystring` module on `npm` that I thought this was in reference to (that didn't have type information).

I have a list of follow-up actions that I will complete after all of the Typescript related PRs have landed.  I will add this to that list, and I think it would be easiest to update this at that time to avoid merge conflicts with the upcoming PRs.